### PR TITLE
Feature/pelagos 4762 update symfony and recipies

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -15,18 +15,18 @@
 
 ###> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=7fd781d3c8b25811ddd82260d88cc496
-#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
-#TRUSTED_HOSTS='^localhost|example\.com$'
+APP_SECRET=ef969998df1caf6e12429ee6e822a47b
```
Comment (mickel1138) [inline .env:18]:
```suggestion
APP_SECRET=7fd781d3c8b25811ddd82260d88cc496
```

**Review by praneethpr — APPROVED** [review]

File: config/packages/prod/deprecations.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# As of Symfony 5.1, deprecations are logged in the dedicated "deprecation" channel when it exists
+#monolog:
+#    channels: [deprecation]
+#    handlers:
+#        deprecation:
+#            type: stream
+#            channels: [deprecation]
+#            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
```
Comment (fightingtexasaggie) [inline config/packages/prod/deprecations.yaml:8]: Why is this commented out? Isn't this replacing the deprecated stuff removed from the config/packages/<mode>/monolog.yaml?

**Review by fightingtexasaggie — COMMENTED** [review]
Do any of these in src/form also need csrf_protection set to false:

DatasetSubmissionXmlFileType.php
LoginForm.php
ReportFundingORganizationType
DatasetType.php
DatasetSubmissionType.php

File: config/packages/prod/deprecations.yaml (hunk 2 above)
Comment (mickel1138) [reply config/packages/prod/deprecations.yaml:8]: This is all on by default

File: .env [hunk 3]
```diff
@@ -16,17 +16,17 @@
...
 
 ###> doctrine/doctrine-bundle ###
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
-DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=14&charset=utf8"
+# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7&charset=utf8mb4"
+DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply .env:29]: Just an example

**Review by fightingtexasaggie — APPROVED** [review]
I think it's ready to go.

File: config/packages/prod/deprecations.yaml (hunk 2 above)
Comment (mickel1138) [reply config/packages/prod/deprecations.yaml:8]: It may not be on by default, but I think it's ok for now. We can decide if we even need this.

**Review by praneethpr — APPROVED** [review]

**Pull request merged**